src/templates/emails/summary_sample_event.html

```diff
@@ -0,0 +1,12 @@
+{% extends "emails/base.html" %}
+{% block title %}Summary Sample Event Report{% endblock %}
```

Comment:
{% block title %}{human readable protocol used in tabs e.g. 'Benthic PIT'} Report{% endblock %}